src/pin-code/pin-code.js

```diff
@@ -128,8 +140,7 @@ export default class PinCode extends React.Component<PropsT, StateT> {
               positive={this.props.positive}
               required={this.props.required}
               size={this.props.size}
-              type="text"
```

Comment:
We can set the type here like so:
```
type={typeof this.props.mask === 'boolean' && this.props.mask ? 'password' : 'text'}
```

Reply:
If we pass it this way, the <MaskToggleShowIcon> will appear.

Reply:
Oh okay that makes sense.